doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
+    In the SQL shell, you'll be connected to a [pre-installed `default` cluster](/sql/show-clusters/#pre-installed-clusters) from which you can get started.
 
-1. **Create secrets.** Secrets allow you to store sensitive credentials securely in Materialize. To retrieve the access credentials, navigate to the [Materialize UI](https://cloud.materialize.com/showSourceCredentials) and replace the placeholders below with the provided values.
+1. Use the [`CREATE CLUSTER`](/sql/create-cluster/) command to create two new clusters, each with a single replica:
 
     ```sql
-    CREATE SECRET kafka_user AS '<KAFKA-USER>';
-    CREATE SECRET kafka_password AS '<KAFKA-PASSWORD>';
-    CREATE SECRET csr_user AS '<CSR-USER>';
-    CREATE SECRET csr_password AS '<CSR-PASSWORD>';
+    CREATE CLUSTER ingest_qck REPLICAS (r1 (SIZE = '2xsmall'));
     ```
 
-1. **Create connections.** Connections describe how to connect and authenticate to an external system you want Materialize to read data from. In this case, we want to [create a connection](/sql/create-connection/#kafka) to a Kafka cluster.
+    ```sql
+    CREATE CLUSTER compute_qck REPLICAS (r1 (SIZE = '2xsmall'));
+    ```
+
+    The `2xsmall` replica size is sufficient for the data ingestion and computation in this getting started scenario. Also, these replicas use only 1 of the [4 credits per hour](https://materialize.com/pricing/) you get with your free trial. Combined with the `xsmall` replica in the pre-installed `default` cluster, which uses 1 credit per hour, you still have 2 hourly credits available.
+
+1. Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the replicas:
 
     ```sql
-    CREATE CONNECTION kafka_connection TO KAFKA (
-      BROKER 'pkc-n00kk.us-east-1.aws.confluent.cloud:9092',
-      SASL MECHANISMS = 'PLAIN',
-      SASL USERNAME = SECRET kafka_user,
-      SASL PASSWORD = SECRET kafka_password
-    );
+    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    ```
+
+    ```noftm
+       cluster   | replica |  size   | ready
+    -------------+---------+---------+-------
+     compute_qck | r1      | 2xsmall | t
+    (2 rows)
     ```
 
-    In addition to Kafka, we need a connection to a Schema Registry that Materialize will use to fetch the schema of our sample data.
+    Once the `r1` replica is ready (`ready=t`), move on to the next step.
+
+## Step 3. Ingest streaming data
+
+Materialize supports streaming data from multiple external sources, including [Kafka](/sql/create-source/kafka/) and [PostgreSQL](/sql/create-source/postgres/). The process for integrating a source typically involves configuring the source's network and creating connection objects in Materialize.
+
+For this guide, you'll keep it simple and use a [built-in load generator](/sql/create-source/load-generator/#auction) that simulates an auction house, where users bid on an ongoing series of auctions.
```

Comment:
```suggestion
For this guide, you'll use a [built-in load generator](/sql/create-source/load-generator/#auction) that simulates an auction house, where users bid on an ongoing series of auctions.
```

Reply:
I feel like I need this to create a contrast to the typical process mentioned prior.